ecommerce-demo/workspaces/childDemo_clothing/workspaceSettings.json

```diff
@@ -0,0 +1,178 @@
+{
+  "colorPalette": {
+    "colorPaletteId": "demo_modern",
```

Comment:
Should we change `"demo_modern"` → `"demo_colorPalette_modern"` to be consistent with the naming for the theme id (at line 72), with has a value of `"demo_theme_xxx"`.